### ISSUE TITLE
Docs are confusing about USWDS's dependency on jQuery

### ISSUE THREAD

**Issue by toolness** [uswds/uswds-site#340]
`developers.md` currently states the following:

> ### Using npm
>
> Note: Using npm to install the Standards will include jQuery version `2.2.0`. Please make sure that you’re not including any other version of jQuery on your page.

Based on the USWDS's [`package.json`](https://github.com/18F/web-design-standards/blob/develop/package.json), it seems jQuery is only mentioned in `devDependencies`.  So at the very least, installing USWDS with `npm install --production` seems like it would not install jQuery.

I am unclear on whether this documentation is actually out of date or not, though. I found #71 and https://github.com/18F/web-design-standards/issues/1351, which are confusing.

It makes sense to me that our *tests* use jQuery while USWDS itself does not; however, it seems inconvenient to then require all developers who use a node-based toolchain to bundle their assets to then be required to use jQuery 2.2.0 in their project.  It's also unclear as to whether they'll only have to use this version of jQuery during development vs. in production.

In short, as someone new to USWDS, I find this jQuery business *extremely* confusing. If this documentation is not out of date, I think we should expand it to include a bit of rationale on *why* jQuery is included when using npm but *not* when including the pre-built `uswds.js`.

**Comment by toolness** [comment]
@msecret do you have any insight on this?

**Comment by msecret** [comment]
I thought jQuery was only required for the tests, not in the actual JS code, so therefore isn't required on production at all.

**Comment by toolness** [comment]
But then shouldn't that mean that it's OK for developers to include their own version of jQuery on their page, if they want?

**Comment by msecret** [comment]
Yes they definitely can

**Comment by maya** [comment]
Seems like the reference to jQuery was removed from the page, so this can be closed.
